### PR TITLE
Refactor code 

### PR DESCRIPTION
**Pull request by kenz-hr**
hi, i'm student, i made some modifications on your code as part of my software engineering project

### CODE REVIEW

**Comment by kenz-hr** [comment]
Refactor code

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/net/NetApplet.java [hunk 1]
```diff
@@ -228,6 +207,31 @@ public String getAppletInfo() {
                + "\nThis software is licensed under LGPL or"
                + "\nCreative Commons 3.0 BY";
     }
+
```
Comment (wumpz) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/net/NetApplet.java:210]: Why is this replaced?

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/net/NetPanel.java [hunk 2]
```diff
@@ -76,32 +83,39 @@ public NetPanel() {
...
-        group.add(rbmi);
-        m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 2, null)));
-        group.add(rbmi);
-        m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 3, null)));
-        group.add(rbmi);
-        m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 4, null)));
-        group.add(rbmi);
+        m.add(rbmi1);
+        group.add(rbmi1);
+        JRadioButtonMenuItem rbmi2 = new JRadioButtonMenuItem(new ZoomAction(editor, 0.25, null));
```
Comment (wumpz) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/net/NetPanel.java:91]: Why those variables? (rbmi3)

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/odg/ODGApplicationModel.java [hunk 3]
```diff
@@ -165,7 +165,7 @@ private void addStrokeButtonsTo(JToolBar bar, DrawingEditor editor) {
      */
     @Override
     public java.util.List<JToolBar> createToolBars(Application a, View pr) {
-        ResourceBundleUtil labels = ResourceBundleUtil.getBundle("org.jhotdraw.draw.Labels");
+        ResourceBundleUtil LABELs = ResourceBundleUtil.getBundle(LABEL);
```
Comment (wumpz) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/odg/ODGApplicationModel.java:168]: When renaming to a constant, then complete uppercase.

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/odg/ODGApplicationModel.java [hunk 4]
```diff
@@ -200,11 +200,11 @@ public void initView(Application a, View p) {
     @Override
     public ActionMap createActionMap(Application a, View v) {
         ActionMap m = super.createActionMap(a, v);
-        ResourceBundleUtil drawLabels = ResourceBundleUtil.getBundle("org.jhotdraw.draw.Labels");
+        ResourceBundleUtil drawLABELs = ResourceBundleUtil.getBundle(LABEL);
         AbstractAction aa;
         m.put(ExportFileAction.ID, new ExportFileAction(a, v));
         m.put("view.toggleGrid", aa = new ToggleViewPropertyAction(a, v, ODGView.GRID_VISIBLE_PROPERTY));
-        drawLabels.configureAction(aa, "view.toggleGrid");
+        drawLABELs.configureAction(aa, "view.toggleGrid");
```
Comment (wumpz) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/odg/ODGApplicationModel.java:207]: strange naming

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/svg/SVGView.java [hunk 5]
```diff
@@ -50,6 +50,9 @@
      */
     private UndoRedoManager undo;
     private PropertyChangeListener propertyHandler;
+    // Variables declaration - do not modify//GEN-BEGIN:variables
```
Comment (wumpz) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/svg/SVGView.java:53]: Why all those `// Variables declaration - do not modify//GEN-BEGIN:variables`? Where does this come from?

**Review by wumpz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**